src/index.html

```diff
@@ -6,9 +6,91 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+
+    <header class="header">
+
+      <div class="shell">
+
+        <div class="header__shell">
+
+          <a href="#" class="logo">
+
+            <img
+              src="images/logo.png"
+              alt="Moyo logo"
+              >
+          </a>
+
+          <nav class="header__nav">
+
+            <ul class="nav">
+
+              <li class="nav__item">
```

Comment:
There are too much blank lines
It's great to visually separate some big code blocks with blank lines, it's increase readability, but in this case it works opposite 🥲